changelog.md

```diff
@@ -5,6 +5,20 @@ _Nothing Yet!_
 
 ## 2018-01-09 - Prop tweaks, including fix for banner-alert when `show` is `false`
 
+**Breaking:**
+- bpk-component-datepicker:
+- bpk-component-popover:
+- bpk-component-tooltip:
+  - Swapped out tether positioning library in favour of popper.js
+  - Removed `tetherOptions` prop
+  - Added `placement` prop, accepts either `top`, `right`, `bottom` or `left`
+
+- bpk-component-popover:
+  - On mobile screen sizes, popovers now have a margin to prevent them from filling the entire width of their container.
+
+- bpk-mixins:
+  - TODO: I think we should just YOLO this and not make it breaking.
```

Comment:
So you reckon we keep the changes as-is, but don't call it a breaking change?

Reply:
Ah so yeah - I highly doubt that folk are using the arrow mixins.

Reply:
Yeah that sounds good to me, let's yolo it